clinica/iotools/converters/nifd_to_bids/nifd_utils.py

```diff
@@ -310,8 +309,8 @@ def convert_dicom(sourcedata_dir: PathLike, bids_filename: PathLike) -> None:
 
     # Run conversion with dcm2niix with anonymization and maximum compression.
     run_dcm2niix(
-        input_dir=sourcedata_dir,
-        output_dir=output_dir,
+        input_dir=str(sourcedata_dir),
+        output_dir=str(output_dir),
```

Comment:
Why do you need to cast to string here? `output_dir` should already be a string after casting at line 302, right?